.github/codecov.yml

```diff
@@ -0,0 +1,43 @@
+# Reference docs at:
+# https://docs.codecov.io/docs/codecovyml-reference
+
+codecov:
+  require_ci_to_pass: yes
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+  status:
+    project:
+      default:
+        # check only master and branches that should be merged directly into
+        # master
+        branches:
+          - main
+          - develop
+          - "feature/*"
+          - "release/*"
+          - "change_readme"
```

Comment:
Remove all the restrictions on branches, we are not following git flow model in this repository.

Reply:
Most likely, you can just kill the `status` key, but check for it.